src/types.d.ts

```diff
@@ -5,13 +5,16 @@ interface CustomDataProvider {
 
 type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>
 
+type UserRole = 'rco-user' | 'rco-power-user'
+
```

Comment:
@salony75way - we have `rco-user` and `rco-power-user` written as strings in two different files.  As soon as I see this, I think _these should go into the constants files_ :-)

Reply:
@IanMayo  This is the type file for user roles

Reply:
Aah yes, my mistake.  I was worried that we may be able to change `rco-power-user` to `rco-super-user` above, and the change not be spotted in `options.ts`.  But, I see in `options.ts` that it is declared of type `UserRole` - so the compiler would spot the error.